### PR TITLE
[Feat: #152350573] Redesign header with Material-UI

### PR DESCRIPTION
**Pull request by fisayomi**
#### What does this PR do?
- Redesign Header
- Enable responsiveness on Chrome

#### How should this be manually tested?
-yarn install (For installation of dependencies)
-yarn start (To fire up webpack server)

#### What are the relevant pivotal tracker stories?
[#152350573](https://www.pivotaltracker.com/n/projects/2091604/stories/152350573)

#### Screenshots (if appropriate)
screen shot 2017-11-17 at 12 04 18 pm
screen shot 2017-11-17 at 12 05 01 pm
screen shot 2017-11-17 at 12 07 41 pm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "bootstrap": "^3.3.7",
     "cheerio": "^1.0.0-rc.2",
     "css-loader": "^0.28.7",
+    "debounce": "^1.1.0",
```
Comment (omorilewa) [inline package.json]: Ma, Is there any reason we have this package here?

File: package.json [hunk 2]
```diff
@@ -41,14 +42,15 @@
...
+    "material-ui": "^0.19.4",
     "mocha": "^3.5.0",
     "node-sass": "^4.5.3",
     "npm-run-all": "^4.0.2",
     "path": "^0.12.7",
     "prop-types": "^15.5.10",
     "react": "^16.0.0",
     "react-apollo": "^1.4.15",
-    "react-bootstrap": "^0.31.3",
+    "react-bootstrap": "^0.31.5",
```
Comment (omorilewa) [inline package.json]: Also, Are e still using this?

Comment (fisayomi) [reply package.json]: yes, it should be uninstalled once we change the signin/signup forms @omorilewa

**Comment by omorilewa** [comment]
@TUgonna @oreoluwade Is there anything in particular that is delaying us from approving this PR. I think it has lingered for too long. cc @fisayomi

**Pull request merged**